import_cli.adoc

```diff
@@ -0,0 +1,183 @@
+[#importing-a-managed-cluster-with-the-cli]
+= Importing a managed cluster with the CLI
+
+After you install multicluster engine for Kubernetes, you are ready to import a cluster to manage.
+Using the OpenShift Container Platform CLI, you can import a cluster using the `kubeconfig` file of the managed cluster. Alternatively, you can run import commands manually on the managed cluster.
```

Comment:
Sug: "...of the managed clsuter." -> "of the cluster to be managed."  (2x) (This is more of the idea that the cluster isn't really a managed cluster until *after* its imported.)

Reply:
Maybe "cluster for import" or "cluster to import"?

Reply:
Or "import cluster" ?